Contributions.md

```diff
@@ -0,0 +1,101 @@
+# Contribution Guide
+
+## Building
+
+Simply run `make install` - this will compile the provider and install it to `~/.terraform.d`. When running this, it will take the current tag and bump it 1 patch version. It does not actually create a new tag (that will be `make release`). If you wish to use the locally installed provider, make sure your TF script refers to the new version number.
+
+Requirements:
+- [Terraform](https://www.terraform.io/downloads.html) 0.13+
+- [Go](https://golang.org/doc/install) 1.18+ (to build the provider plugin)
+
+## Debugging
+
+See [debugging wiki](https://github.com/jfrog/terraform-provider-artifactory/wiki/Debugging).
+
+## Testing
+
+First, you need a running instance of the JFrog Artifactory.
+
+You can ask for an instance to test against it as part of your PR. Alternatively, you can run the file [scripts/run-artifactory.sh](scripts/run-artifactory.sh).
```

Comment:
Please add that the user can run one instance of Artifactory with ./run-artifactory-container.sh, without using docker-compose.